docs/integrations/sources/incident-io.md

```diff
@@ -0,0 +1,59 @@
+# Incident.io
+
+This is the setup guide for the Incident.io source connector which ingests data from the incident API.
+
+## Prerequisites
+
+An API key is required for authentication and using this connector. In order to obtain an API key, you must first create an Incident.io account.
+You can create an account here https://incident.io/
```

Comment:
Linking to the vendor home page without clarifying how to navigate to the page where I can make a token is not good enough.